portfolio/src/main/webapp/index.html

```diff
@@ -9,12 +9,18 @@
   <body>
     <div id="content">
       <h1>Kenneth Kenfack's Portfolio</h1>
-      <p>This is my portfolio. It's all about me a Cameroonian king. Growing up to be amazing. I'm currently a sophmore at umbc</p>
-      <p> I love going on adventures, trying new things, track and field, video games, basketball </p>
+      <p>This is portfolio is all about me a Cameroonian king. How I am growing up to be amazing. I'm currently a sophmore at UMBC</p>
+      <p>I love going on adventures, trying new things, track and field, video games, basketball</p>
+      
+      <p>Instagram @:kenneth_kenfack<br/>
+          LinkedIn: <a href="https://www.linkedin.com/in/kenneth-kenfack-mba-3a003318a/"> Click Me </a>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
       <img src = "/images/IMG_2768rot.jpg"/>
+      <br>
+      <button onclick = "getData()"> Get Data!</button>
```

Comment:
Per my previous comments: you should avoid spaces between attribute, `=` and its value as I would not expect all UAs to parse it properly.